app/assets/stylesheets/navbar.scss

```diff
@@ -21,5 +21,5 @@
   text-align: center;
   padding: 8px;
   margin: 12px 0;
-  background-color: rgba(238, 49, 36, 0.5);
+  //background-color: rgba(238, 49, 36, 0.5); // Set to standard Bootstrap colors
```

Comment:
In general, it's almost always better to delete code rather than leave it in, but commented out.